assets/styles/cookie-consent.scss

```diff
@@ -192,21 +161,72 @@
     text-align: center;
     vertical-align: top;
     color: #555;
-    font-size: 18px;
+    font-size: 20px;
+  }
+
+  &:checked {
+
+    & + .cookie-consent-option__label {
+      font-weight: normal;
+
+      &:before {
+        padding: 2px;
+        content: '\2713';
+        color: var(--cookie-notice-checkbox-color);
+      }
+    }
+
+    &:disabled + .cookie-consent-option__label:before {
+      border: 2px solid currentColor;
+      color: #aaa;
```

Comment:
Why not using `--cookie-notice-button-color-hover: #aaa;`?

Reply:
Well I know that it's the same hex value but those are two different uses. This `#aaa` hex is for the always-on necessary cookies checkbox. That will never change.

But the custom property you suggest is for the buttons underneath which could/will change based on the client colour palette.